python-lib/dku_google/gcloud.py

```diff
@@ -55,7 +56,10 @@ def _run_cmd(cmd=None, **kwargs):
     rv = p.wait()
     if rv != 0:
         raise Exception(err)
-    return out.rstrip()
+
+    cmd_out = out.rstrip()
+    logging.info(f"'{cmd_pretty}' output: {cmd_out}")
```

Comment:
Just in case the cmd out contains secrets, we need to ensure that the customer knowingly changes the log level.
```suggestion
    logging.debug(f"'{cmd_pretty}' output: {cmd_out}")
```